compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+''''exec "python3" "$0" "$@" #'''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+
+
```

Comment:
You mean that make -h option on onecc-docker with argparse library?
I think that -h option is duplicated with onecc's -h option.
Should we make new flag for getting onecc docker's arguments?

Reply:
I think `onecc-docker` can take own argument. For example, if the user want to pass a personal access token to the request headers, `onecc-docker` should take a token value from argument. In view of this case, this script should implement a separate argparser.

Reply:
If `onecc` is already installed when running `onecc-docker`, shows `onecc` help message and if `onecc` is not installed, shows `onecc-docker` help message. What about this solution?

First of all, let's put the PR without considering `-h` option handling.

Reply:
> If onecc is already installed when running onecc-docker, shows onecc help message and if onecc is not installed, shows onecc-docker help message. What about this solution?

Yes, after finalizing existing developments, it would be a good idea to consider them!

>First of all, let's put the PR without considering -h option handling.

Okay.